docs/management_api.yml

```
@@ -414,7 +414,7 @@ components:
       properties:
         path:
           type: string
-          description: The path of the file on the device to download
+          description: The path of the file on the device after upload
```

The description could be a little more concise, don't you think?
```suggestion
          description: The destination path on the device
```

---

Much better!